lib/beaker-hostgenerator/data/vmpooler.rb

```diff
@@ -35,6 +35,10 @@ module Vmpooler
           'platform' => 'el-7-x86_64',
           'template' => 'centos-7-x86_64'
         },
+        'cumulus25-64' => {
+          'platform' => 'cumulusvx-25-x86_64',
```

Comment:
Current convention for platform seems to be osname-majorversionnumber-arch. @waynr, correct me if I'm wrong